Task II - Header.h

```diff
@@ -0,0 +1,46 @@
+#include <vector>

```

Comment:
Comment about the file itself, not the `#include`: Since this header file is all about the `Store` class, a better name would be `Store.h`.  Maybe you needed to name it the way you did because your instructor or grader required it.  That's OK, but the common way to name header files is for their class.  And it would usually have a `Store.cpp` to go along with it.